ui/admin/app/controllers/scopes/scope/session-recordings/session-recording/channels-by-connection/index.js

```diff
@@ -33,4 +44,42 @@ export default class ScopesScopeSessionRecordingsSessionRecordingChannelsByConne
       this.model?.sessionRecording?.state === STATE_SESSION_RECORDING_UNKNOWN
     );
   }
+
+  // =actions
+
+  /**
+   * Reapplies storage policy dates to session recording
+   * @param {SessionRecordingModel}
+   */
+  @action
+  @notifyError(({ message }) => message, { catch: true })
+  @notifySuccess(({ end_time }) =>
+    !end_time
+      ? 'resources.policy.messages.later'
+      : 'resources.policy.messages.reapply',
+  )
+  async reapplyStoragePolicy(sessionRecording) {
+    try {
+      await sessionRecording.reapplyStoragePolicy();
+    } catch (e) {
+      await this.router.replaceWith('scopes.scope.session-recordings');
+      this.router.refresh();
+      throw new Error(e);
+    }
+  }
+
+  /**
+   * Deletes the session recording
+   * @param {SessionRecording} recording
+   */
+  @action
+  @loading
+  @confirm('resources.session-recording.questions.delete')
+  @notifyError(({ message }) => message, { catch: true })
+  @notifySuccess('notifications.delete-success')
+  async delete(recording) {
```

Comment:
non-blocking: In both actions we are referring to session recordings but name them differently. Could we use the same name for consistency now that these are in the same file.

Reply:
yes, great catch!